### ISSUE TITLE
Distance: Update Dynamic Description of the Mass Spheres

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab-basics#94]
From https://github.com/phetsims/gravity-force-lab-basics/issues/86

**The Dynamic Description of the Mass Spheres**
Note that in the default state, the list in REGULAR has 3 bullets and in BASICS the list has 4 bullets. Different states of this list are explained in the design document. Here's one example for the default state with 3 bullets for REGULAR: 
- Change the following 3 bullets in REGULAR from
   - [x] Force on ‪m1‬ by ‪m2‬ is {{tiny}}, and vector points directly at ‪m2‬.
   - [x] Force vector magnitude is {{0.041712 micronewtons}}.
   - [x] m1‬ is very small at {{50 kilograms}} and at {{2.8 ‪meter‬ mark}}.
- To 4 bullets in BASICS
  - [x] Force arrow is {{tiny}}, and points directly at mass 2.
  - [x] Size of force on mass 1 by mass 2 is {{33.4 Newtons}}.
  - [x] Mass 1 is {{2 billion kilograms}}.
  - [x] Mass 1 is at {{2800 kilometer mark}}{{, {{4 kilometers}} from mass 2}}.

I thought it deserved its own issue.

**Comment by zepumph** [comment]
needs to be done before interview start

**Comment by zepumph** [comment]
I didn't have time to tackle this today, but will look at it first thing next week.

**Comment by zepumph** [comment]
@terracoda this is ready for review. At this point I would expect to see zero uses of "vector" or "m1" in basics.

**Comment by terracoda** [comment]
@zepumph, sounds good! Closing.

**Comment by terracoda** [comment]
@zepumph, apologies I may want to revisit this dynamic list. I am keeping this assigned to myself for now, but this is what I am thinking:
Will 3 bullets in BASICS work like this

**With Distance checked**
- Force arrow is {{tiny}}, and points directly at mass 2. 
- Size of force on mass 1 by mass 2 is {{33.4 Newtons}}.
- **Mass 1 is {{2 billion kilograms}} and {{4 kilometers}} from mass 2}}.**

**With Distance not checked**
- Force arrow is {{tiny}}, and points directly at mass 2. 
- Size of force on mass 1 by mass 2 is {{33.4 Newtons}}.
- **Mass 1 is {{2 billion kilograms}} and {{close to mass 2}}.**

**Comment by terracoda** [comment]
There are 2 screen shots in issue #101 that show all the distance descriptions changes that happen in the PDOM when Distance is checked and unchecked. 

Issue #100 links together three issues: this one (#94), #100, and #101.

**Comment by zepumph** [comment]
I implemented this above. In the photos, there were commas between the mass/size and the distance/position sections. I noticed above there wasn't a comma, so I removed the comma. Let me know what you think.

**Comment by terracoda** [comment]
Oops, sorry, the photos were correct. I would like the commas between mass and distance descriptions.

**Also note**, please add a period at end of third bullet.

I updated the examples in comment https://github.com/phetsims/gravity-force-lab-basics/issues/94#issuecomment-465741105

**Comment by terracoda** [comment]
@zepumph back to you for these minor changes.

**Comment by zepumph** [comment]
comma and period added above.

**Comment by terracoda** [comment]
That was quick! Verified and closing.
